### PR TITLE
[WIP] Updating eventing and eventing-sources API references

### PR DESCRIPTION
**Pull request by nachocano**
Fixes https://github.com/knative/docs/issues/1284

## Proposed Changes

- Re-generating eventing and eventing-sources API references with following env variables:
   * KNATIVE_EVENTING_COMMIT=e9e81345ae8f600b03b95a137c445b2c24937893
   * KNATIVE_EVENTING_SOURCES_COMMIT=094dea9c8e66adc4b81b537e871bcda8c8826691
- These commits are the latest HEAD as of 05/10/2019 @ 3:00pm PST.

### CODE REVIEW

**Comment by nachocano** [comment]
fyi @n3wscott @vaikas-google @Harwayne @grantr @akashrv 

Scott I think your changes are covered with this. Can you take a look at  #1284 and maybe add more details in the issue description?

File: docs/reference/eventing/eventing-sources.md [hunk 1]
```diff
@@ -12,10 +12,6 @@ Resource Types:
 <ul><li>
 <a href="#AwsSqsSource">AwsSqsSource</a>
 </li><li>
-<a href="#ContainerSource">ContainerSource</a>
```
Comment (n3wscott) [inline docs/reference/eventing/eventing-sources.md]: These were not deleted from the api group.

File: docs/reference/eventing/eventing.md [hunk 2]
```diff
@@ -1625,7 +1891,803 @@ string
 </tbody>
 </table>
 <hr/>
+<h2 id="sources.eventing.knative.dev">sources.eventing.knative.dev</h2>
+<p>
+<p>Package v1alpha1 contains API Schema definitions for the sources v1alpha1 API group</p>
+</p>
+Resource Types:
+<ul><li>
+<a href="#ApiServerSource">ApiServerSource</a>
```
Comment (n3wscott) [inline docs/reference/eventing/eventing.md]: ah cool, the are added here. No problem then.

**Comment by n3wscott** [comment]
I think it is out of date, can you rerun?

**Comment by nachocano** [comment]
Yes! Give me few minutes and I'll rerun it

**Comment by nachocano** [comment]
> can you rerun?

Done!

**Comment by nachocano** [comment]
We will have to update the script if we want the eventing-sources contribs to be included in the eventing-sources API references. Otherwise just GitHub is.

**Comment by nachocano** [comment]
@grantr will take a look at this. As this is outdated, I'll just close it for now

**Pull request closed without merging**